qemu/tests/hotplug_mem_repeat.py

```diff
@@ -35,12 +36,17 @@ def start_test(self):
         memory 256 times, then unplug 256 times. Otherwise, repeat hotplug
         and unplug in turn for 256 times. This is test entry.
         """
-        times = int(self.params["repeat_times"])
+        times = self.params.get_numeric("repeat_times", int)
+        get_basic_page = self.params.get("get_basic_page", False)
```

Comment:
```suggestion
        get_basic_page = self.params.get("get_basic_page")
```
According to the naming of the `get_basic_page`, the default value should not be a boolean type.
And I suggest moving it to the blocker `if self.params.get("vm_arch_name", "") == "aarch64":`

Reply:
Good check
Indeed, the default type of this name should not be a boolean type.